docs/connect.html

```diff
@@ -576,6 +578,10 @@ <h4><a id="connect_administration" href="#connect_administration">Kafka Connect
     In most cases, connector and task states will match, though they may be different for short periods of time when changes are occurring or if tasks have failed. For example, when a connector is first started, there may be a noticeable delay before the connector and its tasks have all transitioned to the RUNNING state. States will also diverge when tasks fail since Connect does not automatically restart failed tasks. To restart a connector/task manually, you can use the restart APIs listed above. Note that if you try to restart a task while a rebalance is taking place, Connect will return a 409 (Conflict) status code. You can retry after the rebalance completes, but it might not be necessary since rebalances effectively restart all the connectors and tasks in the cluster.
     </p>
 
+    <p>
+    Starting with 2.5.0, Kafka Connect is using the <code>status.storage.topic</code> to also store information related to the topics that each connector is using. Connect workers use these per connector topic status updates to respond to requests to the REST endpoint <code>GET /connectors/{name }/topics</code> by returning the set of topic names that a connector is using. A request at any time to the REST endpoint <code>PUT /connectors/{name}/topics/reset</code> resets the set of active topics for a connector and allows a new set to be populated based on the connector's latest pattern of topic usage. Upon connector deletion, the set of the connector's active topics is also deleted. Topic tracking is enabled by default but can be disabled by setting <code>topic.tracking.enable=false</code>. If it's desirable to just disallow requests to reset the active topics of connectors during runtime, a user may set the worker property <code>topic.tracking.allow.reset=false</code>.
```

Comment:
Stylistic nits, but not a big deal:

* I think `Kafka Connect is using` -> `Kafka Connect uses` sounds more natural
* I would capitalize `Connect workers` -> `Connect Workers` (partly just because `Worker` is a class name), though it seems we don't consistently capitalize anyway...
* Nit that may not even show up rendered: extra ' ' in `{name }`.

Reply:
Thanks @ewencp! All accepted :)